.github/workflows/stylebot.yaml

```
@@ -0,0 +1,67 @@
+  fix:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          app-id: ${{ secrets.STYLEBOT_GITHUB_APP_ID }}
+          private-key: ${{ secrets.STYLEBOT_GITHUB_APP_KEY }}
+      - uses: actions/checkout@master
+      - id: nodeversion
+        run: echo "version=$(grep nodejs .tool-versions | sed -e 's/[^[:space:]]*[[:space:]]*//')" >> $GITHUB_OUTPUT
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:90: Double quote to prevent globbing and word splitting

(shellcheck)